src/main/drivers/accgyro_fake.c

```diff
@@ -28,26 +50,55 @@
 #include "accgyro.h"
 #include "accgyro_fake.h"
 
-
 static int16_t fakeGyroADC[XYZ_AXIS_COUNT];
 
+#if defined(SIMULATOR_BUILD) && defined(SIMULATOR_MULTITHREAD)
```

Comment:
Yes, certainly your initial goal of making the changes minimal was a good one. Now that it's working we take some bits of it into the mainstream.